src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ExternalReferencesTableNode.cs

```diff
@@ -97,14 +97,14 @@ public override ObjectData GetData(NodeFactory factory, bool relocsOnly = false)
 
             foreach (SymbolAndDelta symbolAndDelta in _insertedSymbols)
             {
-                if (factory.Target.Abi == Internal.TypeSystem.TargetAbi.CoreRT)
+                if (factory.Target.Abi == TargetAbi.CoreRT)
                 {
                     // TODO: set low bit if the linkage of the symbol is IAT_PVALUE.
-                    builder.EmitPointerReloc(symbolAndDelta.Symbol, symbolAndDelta.Delta);
+                    builder.EmitReloc(symbolAndDelta.Symbol, RelocType.IMAGE_REL_BASED_REL32, symbolAndDelta.Delta);
```

Comment:
I think it would look better to use `IMAGE_REL_BASED_RELPTR32` here instead.

To save the hassle with updating the ObjectWriter, it can be special cased in EmitSymbolRef for now: map it to `IMAGE_REL_BASED_REL32` and add sizeof(int) to delta.